builtin_string.go

```diff
@@ -951,8 +988,12 @@ func (r *Runtime) initString() {
 	o._putProp("toString", r.newNativeFunc(r.stringproto_toString, nil, "toString", nil, 0), true, false, true)
 	o._putProp("toUpperCase", r.newNativeFunc(r.stringproto_toUpperCase, nil, "toUpperCase", nil, 0), true, false, true)
 	o._putProp("trim", r.newNativeFunc(r.stringproto_trim, nil, "trim", nil, 0), true, false, true)
-	o._putProp("trimEnd", r.newNativeFunc(r.stringproto_trimEnd, nil, "trimEnd", nil, 0), true, false, true)
-	o._putProp("trimStart", r.newNativeFunc(r.stringproto_trimStart, nil, "trimStart", nil, 0), true, false, true)
+	trimEnd := r.newNativeFunc(r.stringproto_trimEnd, nil, "trimEnd", nil, 0)
+	trimStart := r.newNativeFunc(r.stringproto_trimStart, nil, "trimStart", nil, 0)
+	o._putProp("trimEnd", trimEnd, true, false, true)
+	o._putProp("trimStart", trimStart, true, false, true)
+	o._putProp("trimRight", trimEnd, true, false, true)
+	o._putProp("trimLeft", trimStart, true, false, true)
```

Comment:
TIL `trimLeft` is just an alias for `trimStart`, although I never used `trimStart` before. Double TIL